src/main/kotlin/org/rust/cargo/util/CargoCommandCompletionProvider.kt

```diff
@@ -160,6 +160,8 @@ class CargoCommandCompletionProvider(
 
             targetBench()
             flag("bench")
+
+            flag("all-targets")
```

Comment:
The `targetAll` is used in `doc` and `install` commands, which don't have `--all-targets` flag.